src/instana/instrumentation/aws/s3.py

```diff
@@ -0,0 +1,86 @@
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2020
+
+from typing import TYPE_CHECKING, Any, Callable, Dict, Sequence, Type
+
+from instana.span_context import SpanContext
+
+if TYPE_CHECKING:
+    from botocore.client import BaseClient
+
```

Comment:
Move these lines inside the try-exception.

Reply:
Done